components/extractors/marketing-sales/bing-ads/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Microsoft Advertising (Bing Ads)
+permalink: /components/extractors/marketing-sales/bing-ads/
+redirect_from:
+    - /extractors/marketing-sales/bing-ads/
+---
+
+* TOC
+{:toc}
+
+The Microsoft Advertising (Bing Ads) extractor supports extracting either campaign entity data or various types of reports available. In case of reports you can specify your own set of columns and primary key to use in Keboola Storage, but there are also presets of columns and appropriate primary keys available.
+
+## Prerequisites
```

Comment:
Someone with access to a Microsoft Advertising (Bing Ads) production environment should test this.